### ISSUE TITLE
feature request: dominator tree generation

### ISSUE THREAD

**Issue by thyming** [graphology/graphology#407]

**Comment by thyming** [comment]
Yes, exactly. I ended up implementing the algorithm in [this paper](https://www.cs.rice.edu/~keith/EMBED/dom.pdf
) which is what [networkx in python uses](https://networkx.org/documentation/stable/reference/algorithms/generated/networkx.algorithms.dominance.immediate_dominators.html#networkx.algorithms.dominance.immediate_dominators). If you'd like I can open a PR with it.

**Comment by Yomguithereal** [comment]
> If you'd like I can open a PR with it.

Sure. I guess this should go into a new package in the repo called `dominance` and that will be shipped as `graphology-dominance`? What do you think? Could it be packed into another already existing package?
